content/docs/deployment/csmoperator/modules/replication.md

```diff
@@ -5,23 +5,47 @@ description: >
 
-## Prerequisite
-
-To use Replication, you need at least two clusters:
+## Prerequisites
+To configure Replication prior to installation via CSM Operator, you need:
 
 - a source cluster which is the main cluster
-- one or more target clusters which will serve as diaster recovery clusters for the main cluster
-
-To configure all the clusters, follow the steps below:
+- a target cluster which will serve as the disaster recovery cluster, a single stretched cluster configuration is also supported
+- the csm-replication GitHub repository is cloned to your source cluster and the [`repctl`](../../../../replication/tools) tool is either built from source or downloaded and in your PATH
 
-1. On your main cluster, follow the instructions available in CSM Replication for [Installation using repctl](../../../../replication/deployment/install-repctl), with the exception of step 4. When you reach step 4, you MUST use the command below to automatically package all clusters `.kube` config as a secret:           
+### Cloning the GitHub Repository and Building repctl
+On your source cluster run the following to clone and build the repctl tool.
 
-```shell
-  ./repctl cluster inject 
+```
+git clone -b v1.4.0 https://github.com/dell/csm-replication.git
+cd csm-replication/repctl
+make build
 ```
 
-CSM Operator needs this admin configs instead of the service accounts’ configs  to be able to properly manage the target clusters. The default service account that'll be used is the CSM Operator service account.
-
-2. On each of the target clusters, configure the prerequisites for deploying the driver via Dell CSM Operator. For example, PowerScale has the following [prerequisites for deploying PowerScale via Dell CSM Operator](../../drivers/powerscale/#prerequisite)
+The rest of the instructions will assume that your current working directory is the csm-replication/repctl directory.
+## Configuration Steps
+To configure replication when using multiple clusters:
+
+1. On your main cluster collect the cluster admin configurations for each of the clusters. In the following example the source cluster, `cluster-1` uses configuration `/root/.kube/config-1` and the target cluster, `cluster-2` uses the configuration `/root/.config/config-2`. Use repctl to add the clusters:
+    ```shell
+      ./repctl cluster add -f "/root/.kube/config-1","/root/.kube/config-2" -n "cluster-1","cluster-2"
+    ```
+  > **_NOTE:_**  If using a single Kubernetes cluster in a stretched configuration there will be only one cluster.
+2. Install the replication controller CRDs:
+    ```shell
+    ./repctl create -f ../deploy/replicationcrds.all.yaml
+    ```
+3. Inject the service account's configuration into the clusters. CSM Operator needs the admin configs instead of the service account configurations to be able to properly manage the target clusters.
```

Comment:
I do not understand this step. "Inject" does not seem to be a proper verb here. The step also says to "Inject the service account's configuration...", and then the explanatory text says that admin co0nfigs are needed, not the service account configurations.